README.md

```diff
@@ -3,15 +3,22 @@
 ![Ui](docs/images/Ui.png)
```

Comment:
Could you update the link of the GitHub Actions build status badge in line 1? so that it reflects the build status of this team repo

Reply:
Resolved!